e2e/app/component/base-menu.ts

```diff
@@ -86,7 +86,7 @@ export default abstract class BaseMenu extends Container {
   protected async findMenuItemLink(menuItemText: string, menuParentXpath: string): Promise<Link> {
     const menuItemSelector = `${menuParentXpath}${this.getMenuItemXpath(menuItemText)}`;
     const link = new Link(this.page, menuItemSelector);
-    await link.focus();
+    await link.focus(60000);
```

Comment:
Cohort Criteria Menu could takes longer than 30 seconds to populate du to slower request `GET https://api-dot-all-of-us-workbench-test.appspot.com/v1/cohortbuilder/3/criteria/menu/0`.

Test [failure](https://app.circleci.com/pipelines/github/all-of-us/workbench/9356/workflows/22918bf8-0f0e-4e93-b844-9d615224539b/jobs/133240/tests) stacktrace:

```
Error: TimeoutError: waiting for XPath `//*[contains(concat(" ", normalize-space(@class), " "), " p-tieredmenu ")or contains(concat(" ", normalize-space(@class), " "), " p-menu ")][contains(concat(" ", normalize-space(@class), " "), " p-menu-overlay-visible ")]//*[not(contains(concat(" ", normalize-space(@class), " "), " menuitem-header "))][./*[@role="menuitem" and normalize-space()="Drugs"]]` failed: timeout 30000ms exceeded
    at /home/circleci/workbench/e2e/app/element/base-element.ts:43:15
    at Link.focus (/home/circleci/workbench/e2e/app/element/base-element.ts:251:21)
    at TieredMenu.findMenuItemLink (/home/circleci/workbench/e2e/app/component/base-menu.ts:89:5)
    at TieredMenu.select (/home/circleci/workbench/e2e/app/component/base-menu.ts:44:28)
    at CohortParticipantsGroup.clickCriteriaMenuItems (/home/circleci/workbench/e2e/app/page/cohort-participants-group.ts:132:5)
    at CohortParticipantsGroup.includeDrugs (/home/circleci/workbench/e2e/app/page/cohort-participants-group.ts:97:5)
    at Object.<anonymous> (/home/circleci/workbench/e2e/tests/datasets/dataset-create.spec.ts:38:24)
    at _callCircusTest (/home/circleci/workbench/e2e/node_modules/jest-circus/build/run.js:212:5)
    at _runTest (/home/circleci/workbench/e2e/node_modules/jest-circus/build/run.js:149:3)
    at _runTestsForDescribeBlock (/home/circleci/workbench/e2e/node_modules/jest-circus/build/run.js:63:9)
    ```